shell/server/proxy.js

```diff
@@ -1604,7 +1598,7 @@ Proxy.prototype.handleRequest = function (request, data, response, retryCount) {
     var path = request.url.slice(1);  // remove leading '/'
     var session = self.getSession(request);
 
-    if (request.method === "GET") {
+    if (request.method === "GET" || request.method === "HEAD") {
```

Comment:
Silly (?) question: if `request.method == HEAD`, then shouldn't we do a `session.head()` instead?